pkg/envoy/envoy.go

```diff
@@ -101,24 +160,16 @@ func StartEnvoy(debug bool, adminPort int, stateDir, logDir string, baseID uint6
 			}
 
 			name := "cilium-envoy"
-			logLevel := "info"
-			if debug {
-				// Try first with debug build
-				name = "cilium-envoy-debug"
-				logLevel = "debug"
+			logLevel := envoyLevelMap[log.Level]
+			if logLevel == "" {
+				logLevel = "info"
```

Comment:
maybe set the value via a map lookup, so that the map is the only source of values passed to envoy?

Reply:
Good point! Done for the upcoming build with a rebase to master as well.